lib/geojson.js

```diff
@@ -11,7 +11,8 @@ function GeoJSON(filepath) {
     this.datasource = new mapnik.Datasource({
       type: 'geojson',
       file: filepath,
-      cache_features: false
+      cache_features: false,
+      num_features_to_query: 1000000
```

Comment:
What's the plan with this number? Is it going to be an option when using the library or are we going to calculate it somehow?